src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,90 @@
+import podOverviewChart from './img/pod-overview-chart.png'
+import { ImageMeta } from 'posts/post'
+
+const image: ImageMeta = {
+  url: podOverviewChart,
+  width: 1632,
+  height: 502,
+}
+
+const title =
+  'How our house got our lives back by using a collective “risk budget”'
+
+const author = 'Jeremy Blanchard'
+const date = 'February 7, 2021'
+
+const summary =
+  'Navigating house agreements around COVID can be really stressful. We made a tool to help make it a lot easier.'
+const content = `
+
+<style>
+img {
+  max-width: 100%;
+  height: auto;
+}
+</style>
+
+
+
+*Today we’re launching a new version of the microCOVID Risk Tracker with features for households and pods to collectively budget their risk over time.* [*You can download the latest version of the Risk Tracker here →*](/tracker)
+
+
+<figure>
+  <img src="${podOverviewChart}" alt="Graph of 4 housemate's risk budget, showing risk points used and risk points remaining." />
+</figure>
+
+
+## How the Risk Tracker helped our house get our lives back
+
+Here’s a bit of the backstory behind this new version. Up until [microCOVID](/) was released in August 2020, my house was struggling with the same thing many communal houses were struggling with: we want to make sure to keep ourselves, each other, and the wider community safe from COVID, *and* at the same time, there are things that are important enough that we want to be able to find a way to do them. 
+
+The dilemma: how do you make decisions about what a “safe amount of risk is” when living with many other people?
+
+At that time our system was similar to many other group houses: masked, outdoor, distance hangs were okay, but anything other than that needed group input/consensus. As you might imagine, having every single person’s activities be subject to a consensus discussion in a house meeting is a recipe for a ton of stress and struggle. (“What’s riskier: a 3-hour unmasked. outdoor hang? Or a 30-minute indoor, masked hang? 🤷‍♂️” and “Okay I know you really want to see your bodyworker and I really want to see my partner, but are either of those risks we’re willing to take?!”)
```

Comment:
I'd drop the "as you might imagine" and try to stick with the personal experience.

"Having every single person's activities... _was_ a recipe for a ton of stress and struggle."

Also, in the example I find the "we're willing to take" confusing. I think you mean "are those risks that the people who are not doing the activity are willing to let the other person do the activity." Perhaps:

My housemate, Q really wanted to see their bodyworker. I really wanted to see my partner. But was I ok being in indirect contact with that body worker for Q's sake? Did Q think my happiness from seeing my partner was worth the risk to Q?"

Reply:
Thanks. Yep!